tests/expr_and_series/shift_test.py

```diff
@@ -28,11 +23,8 @@ def test_shift(request: Any, constructor: Any) -> None:
     compare_dicts(result, expected)
 
 
-def test_shift_series(request: Any, constructor_eager: Any) -> None:
-    if "pyarrow_table" in str(constructor_eager):
-        request.applymarker(pytest.mark.xfail)
-
-    df = nw.from_native(constructor_eager(data), eager_only=True)
+def test_shift_series(constructor: Any) -> None:
+    df = nw.from_native(constructor(data), eager_only=True)
```

Comment:
there's been a little renaming going on recently, so we need to change this:
```suggestion
def test_shift_series(constructor_eager: Any) -> None:
    df = nw.from_native(constructor_eager(data), eager_only=True)
```
It needs to be eager because this test checks Series, which is eager-only